resources/tools/mdsd/devops/pipeline/stages/impl/dockerized/maven/modules/Build/Build.groovy

```diff
@@ -1,16 +1 @@
-configFileProvider(
-    [configFile(fileId: 'fba2768e-c997-4043-b10b-b5ca461aff54', variable: 'MAVEN_SETTINGS')]) {
-    extendConfiguration([
-        mavenSettingsFile: MAVEN_SETTINGS,
-        emptySlaveDir: "/tmp/${env.BUILD_TAG}"])
-
-    CFG = getCurrentConfiguration()
-
-    MPLModulePostStep {
-        sh "rm -rf ${CFG.emptySlaveDir}"
-    }
-
-    sh "mkdir -p ${CFG.emptySlaveDir}"
-
-    MPLModule("Build")
-}
\ No newline at end of file
+MPLModule('Prepare Empty Folder For Build')
```

Comment:
This is somewhat confusing. Do I understand the call hierarchy? [Build] --> [Prepare Empty Folder...] --> [Apply Settings...] --> [Build (next in hierarchy)]? I currently do not see why you need this separation. Particularly, if Build only calls PrepareEmptyFolderForBuild. You only use it in the maven modules, right?

Reply:
You are right. I moved the module call to the main module.